packages/shared/src/components/squads/layout/SquadDirectoryLayout.tsx

```diff
@@ -67,17 +67,16 @@ export const SquadDirectoryLayout = (
   return (
     <BaseFeedPage className="relative mb-4 flex-col px-4 pt-4 laptop:px-18 laptop:pt-8">
       <header className="w-full">
-        {isMobileLayout && (
```

Comment:
This shouldn't have been removed right?
On mobile I see a different design:

![Screenshot 2024-09-06 at 11 16 44](https://github.com/user-attachments/assets/e3d97ff7-9cd6-47f5-84ab-a21f2bc2c7de)

Reply:
This was removed due to errors in console, UI from client was different from server :( 

Checking for mobile design but probably didn't implemented, my bad!

Reply:
Yes! Thanks!